src/index.ts

```diff
@@ -260,14 +265,16 @@ export const syncLogger = (logger: Logger, namespace: string): Logger => {
     if (internals.levels) {
         internals.levels.forEach((level, idx) => {
             if (level === 'none') return
-            if (!internals.isEnabled?.(namespace, idx)) {
+            const levelIsEnabled = internals.isEnabled?.(namespace, idx) ?? false
+            if ( levelIsEnabled || canForceWrite) {
+                enabledLevels[level] = levelIsEnabled
+
+                logger[level] = ((contextId: string, message: string, data?: Record<string, unknown>, forceWriteOn?: boolean) => {
```

Comment:
Can we have a more simple solution like: when we recevie a parameter `forceLoging` in logMethod --> enableLevel (with setLevel) or activate `logger.isLevelEnabled` ?

Reply:
I don't know that works or not. But I think we can try it because you always have a very good idea to resolve the solution.

Reply:
I try but is not possible with the actual system. We have no function in logMethod when the level is not enable so we can't call setLevel or others. We need to enable all level at the init of logger.